Governance/introduction.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+This book, '**Climate Dataset Guidelines**', provides guidelines for creating, managing, sharing and publishing climate data. With a focus on Australian climate research it has been designed to enable a common approach across the community. Despite the focus on climate data, many of the principles and tools described within are applicable to many Earth science data applications.
+
+These guidelines are applicable to all levels of data sharing, from peer-to-peer sharing with colleagues to full publication in a public repository. If you expect that even a single other person will use your data for any purpose, including simple checking and comparing, your data should meet FAIR principles and meet a minimum set of formatting and metadata recommendations.

```

Comment:
Could tone this down a little - all of FAIR is a bit much for a single collaborator! :) maybe "If you expect that even one other person will use your data for any purpose <...> your data should be structured appropriately considering aspects of FAIR principles and <...>"